python/ray/_private/runtime_env/context.py

```diff
@@ -42,11 +42,30 @@ def serialize(self) -> str:
     def deserialize(json_string):
         return RuntimeEnvContext(**json.loads(json_string))
 
+    @staticmethod
+    def _fix_windows_args(passthrough_args: List[str]) -> List[str]:
+        """
+        Fixes arguments on Windows if the argument is a path with a space
+
+        Note that the function only works if the file path actually exists.
+
+        Args:
+            passthrough_args: the passthrough arguments
+
+        Returns:
+            Fixed passthrough arguments
+        """
+        for i, arg in enumerate(passthrough_args):
+            if os.path.isfile(arg) or os.path.isdir(arg):
```

Comment:
Why not `if os.path.exists(arg)` ?

Reply:
That makes sense. Updated in the latest commits